app/Community/Actions/BuildTrendingGamesAction.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Actions;
+
+use App\Models\Game;
+use App\Platform\Data\GameData;
+use Illuminate\Support\Collection;
+
+/**
+ * TODO bake more intelligence into this algorithm
+ *
+ * come up with some kind of "trending score" that would combine various facets, such as:
+ * - boost for recent player sessions
+ * - boost for recent achievement unlocks
+ * - boost if game is gaining new unique players
+ * - boost if people are getting beats/masteries
+ * - decay the score every N days the game has been trending. after it isn't trending anymore, stop the decay.
+ */
+
+// TODO allow filtering by system
```

Comment:
I'd like to make this algorithm much more intelligent in the future. Right now isn't a good time for it though - I just want to get this new home page over the finish line.

For now, I'm just throwing my ideas/brainstorming for an improved algorithm in here.